src/containers/GameListPage.js

```diff
@@ -15,6 +18,15 @@ const mapDispatchToProps = dispatch => ({
   },
   joinRoom: (roomId) => {
     dispatch( joinRoom(roomId) );
+  },
+  getPlayers: (rooms) => {
+    dispatch( getPlayers(rooms) );
+  },
+  getRoom: (rooms) => {
+    dispatch( getPlayers(rooms) );
```

Comment:
prop.function name is `getRoom`, but you use `getPlayers` function. (It doesn't match function name.)